models/DomainApimResponse.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  EmailString,
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "@pagopa/ts-commons/lib/strings";
+import * as E from "fp-ts/lib/Either";
+import { withDefault } from "@pagopa/ts-commons/lib/types";
+import * as t from "io-ts";
+import { identity, pipe } from "fp-ts/lib/function";
+
+/**
+ * Check data of a ApimUserResponse to parse if it's an Organization account.
+ * By convention, we decided Organization accounts store the OrganizationFiscalCode as note
+ * Thus, by parsing notes we can infer it's an Organization
+ *
+ * @param u
+ * @returns
+ */
+const isOrganizationUser = (u: RawApimUserResponse): boolean =>
+  pipe(
+    u,
+    E.fromPredicate(
+      () => u.note !== undefined,
+      () => E.toError
+    ),
+    E.map(() => pipe(u.note.trim(), OrganizationFiscalCode.decode, E.isRight)),
+    E.mapLeft(() => false),
+    E.fold(() => false, identity)
+  );
+
+export type RawApimUserResponse = t.TypeOf<typeof RawApimUserResponse>;
+export const RawApimUserResponse = t.interface({
+  email: EmailString,
+  firstName: NonEmptyString,
+  id: NonEmptyString,
+  lastName: NonEmptyString,
```

Comment:
in case of `organization` type, what `lastName`, what `lastName` will contain?

Reply:
I think it's empty at moment, isn't it @balanza?

Reply:
https://github.com/pagopa/io-developer-portal-frontend/blob/5a8e53b96f64fc255714d39fd15c51cfda9e9507/src/pages/Profile.tsx#L241

Reply:
So it's actually a `NonEmptyString`, thanks